dashboard/src/components/IssueDetails/IssueDetails.tsx

```diff
@@ -0,0 +1,144 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+import { ErrorBoundary } from 'react-error-boundary';
+import { useMemo } from 'react';
+
+import SectionGroup from '@/components/Section/SectionGroup';
+import type { ISection } from '@/components/Section/Section';
+import UnexpectedError from '@/components/UnexpectedError/UnexpectedError';
+
+import { shouldTruncate, valueOrEmpty } from '@/lib/string';
+
+import { TruncatedValueTooltip } from '@/components/Tooltip/TruncatedValueTooltip';
+
+import { getMiscSection } from '@/components/Section/MiscSection';
+
+import { useIssueDetails } from '@/api/issueDetails';
+
+interface IIssueDetails {
+  issueId?: string;
+  versionNumber?: string;
+}
+
+export const IssueDetails = ({
+  issueId,
+  versionNumber,
+}: IIssueDetails): JSX.Element => {
+  const { data, error, isLoading } = useIssueDetails(
+    issueId ?? '',
+    versionNumber ?? '',
+  );
+
+  const { formatMessage } = useIntl();
+
+  const miscSection: ISection | undefined = useMemo(():
+    | ISection
+    | undefined => {
+    return getMiscSection({
+      misc: data?.misc,
+      title: formatMessage({ id: 'globalDetails.miscData' }),
+    });
+  }, [data?.misc, formatMessage]);
+
+  const generalSections: ISection[] = useMemo(() => {
+    if (!data) {
+      return [];
+    }
+    return [
+      {
+        title: data.id,
+        eyebrow: formatMessage({ id: 'issueDetails.issueDetails' }),
+        subsections: [
+          {
+            infos: [
+              {
+                title: 'issueDetails.version',
+                linkText: data.version.toString(),
+              },
+              {
+                title: 'global.origin',
+                linkText: valueOrEmpty(data.origin),
+              },
+              {
+                title: 'issueDetails.reportUrl',
+                linkText: shouldTruncate(valueOrEmpty(data.report_url)) ? (
+                  <TruncatedValueTooltip value={data.report_url} isUrl={true} />
+                ) : (
+                  valueOrEmpty(data.report_url)
```

Comment:
```suggestion
                  truncateUrl(data.report_url)
```

Reply:
it will only fall under that else if it shouldn't truncate